test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -208,7 +220,7 @@ UCS_TEST_P(test_uct_sockaddr, connect_client_to_server_with_delay)
 {
     delay_conn_reply = true;
     client->connect(0, *server, 0, m_connect_addr, client_iface_priv_data_cb,
-                    NULL, NULL, &client->max_conn_priv);
+                    NULL, NULL, this);
```

Comment:
does it really need to be test instance?

Reply:
i use it in the pack_cb. what do you mean?

Reply:
it's OK if priv_data is member of test instance...